app/routes/member.js

```diff
@@ -26,10 +26,10 @@ export default class MembersRoute extends AuthenticatedRoute {
     }
 
     model(params) {
-        this._requiresBackgroundRefresh = false;
+        this._requiresBackgroundRefresh = true;
 
         if (params.member_id) {
-            return this.store.queryRecord('member', {id: params.member_id, include: 'email_recipients'});
+            return this.store.queryRecord('member', {id: params.member_id, include: 'email_recipients,products'});
```

Comment:
I've added this here AND to the controller, I'm not sure which is correct, or if we need both